deployment/aws-terraform/application/cloud-carbon-footprint/cloud-carbon-footprint.tf

```diff
@@ -106,8 +106,7 @@ resource "aws_iam_role" "ccf" {
       "Action": "sts:AssumeRoleWithWebIdentity",
       "Condition": {
         "StringEquals": {
-          "${module.eks.oidc_provider}:aud": "sts.amazonaws.com",
-          "${module.eks.oidc_provider}:sub": "system:serviceaccount:daskhub:default"
```

Comment:
You're going to need a `sub` condition; you'll just need to direct it to the right namespace.  (So, change the `daskhub` value to ... `ccf`?  Or whatever you named it.)

Reply:
Ok.  Thanks for that. FIxed.